tests/vm-migration

```diff
@@ -119,15 +136,33 @@ lxc exec member1 -- lxc storage create ceph ceph
 lxc exec member1 -- lxc storage volume create ceph vol1 --type=block size=500MiB
 
 # Create a VM in the cluster, on member1.
-lxc exec member1 -- lxc init "${TEST_IMG:-ubuntu-minimal-daily:24.04}" v1 --vm --storage ceph --target member1 -c migration.stateful=true -c limits.memory=512MiB
+lxc exec member1 -- lxc init "${TEST_IMG:-ubuntu-minimal-daily:24.04}" v1 --vm --storage ceph --target member1 -c migration.stateful=true -c limits.memory=1GiB
 
 # Add vol1 as a disk device to the VM.
 lxc exec member1 -- lxc config device add v1 vol1-disk disk pool=ceph source=vol1
 
 # Start the VM.
 lxc exec member1 -- lxc start v1
-sleep 60
+
+# Wait for a long time for it to boot (doubly nested VM takes a while).
+while [ "$(lxc exec member1 -- sh -c "lxc info v1 | grep -F 'Processes:' | cut -d':' -f2 | tr -d '[:blank:]'")" -le 1 ]; do
```

Comment:
I think the wrapper shell is not needed.

```suggestion
while [ "$(lxc exec member1 -- lxc info v1 | awk '{if ($1 == "Processes:") print $2}')" -le 1 ]; do
```
Also changed to use `awk` like we do elsewhere to get the process count.

Reply:
With awk I think I need the subshell:
```
$ [ lxc exec member1 -- lxc info v1 | awk '{if ($1 == "Processes:") print $2}' -le 1 ]
awk: fatal: cannot open file `-le' for reading: No such file or directory
```

Reply:
Sorry for not being clear. With the subshell, I meant the `sh -c` bit used while doing the `lxc exec`.